src/System.Collections.Concurrent/src/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netcoreapp;
```

Comment:
We should not need `netcoreapp` build configuration now that we have `netcoreapp--Window_NT` and `netcoreapp-Unix` configurations.